Trappist/src/Promact.Trappist.DomainModel/ApplicationClasses/Question/SeedDatabase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Enum;
+using Promact.Trappist.DomainModel.Models.Question;
+using System.Linq;
+
+namespace Promact.Trappist.DomainModel.ApplicationClasses.Question
+{
+
+    /// <summary>
+    /// Polpulates pre-required table
+    /// </summary>
+    public class SeedDatabase
```

Comment:
seed database should be in domain model namespace. Create one folder and add your seed in that folder.